include/xtensor/xmath.hpp

```diff
@@ -582,57 +513,50 @@ XTENSOR_INT_SPECIALIZATION_IMPL(FUNC_NAME, RETURN_VAL, unsigned long long);
 
     namespace math
     {
-        template <class T>
+        template <class T = void>
```

Comment:
Why keeping this template parameter while it is not used anywhere in the class?

Reply:
This one is used in the reducers unfortunately. In fact I might still need to figure out what to do here. (it's used for amax/amin)